internal/config/profile.go

```diff
@@ -226,20 +229,65 @@ func (p *profile) SetOrgID(v string) {
 }
 
 // GetConfigDescription returns a map describing the configuration
-func GetConfigDescription(name string) map[string]string {
-	redactedValue := "redacted"
+func GetConfigDescription(name string, redact bool) map[string]string {
+	settings := viper.GetStringMapString(name)
+	newSettings := make(map[string]string)
+
+	fmt.Printf("key %s\n", settings)
+
+	for k, v := range settings {
+		fmt.Printf("key %v val %v\n", k, v)
+		if redact && (k == privateAPIKey || k == publicAPIKey) {
+			newSettings[k] = "redacted"
+		} else {
+			newSettings[k] = v
+		}
+	}
+
+	return newSettings
+}
+
+// DeleteConfig deletes an existing configuration
+func DeleteConfig(name string) error {
```

Comment:
[big nit] this may have not being obvious at first glance but this files tries to follow a singleton pattern. `p` is your singleton instance so this method should follow the pattern
short suggestion
```go
func Delete() error { return p.Delete() }
func (p *profile) Delete() error {
...  
 t.Delete(&p.name)
...
}
```
would follow the style of this file for this operation